devicon.json

```diff
@@ -3239,6 +3239,29 @@
         "color": "#0769ad",
         "aliases": []
     },
+    {
+        "name": "json",
+        "altnames": [
+            "JavaScript Object Notation"
+        ],
+        "tags": [
+            "format",
+            "standard",
+            "file-format",
+            "object-notation"
```

Comment:
Should we add JavaScript here?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
I think it's fine because that's what JSON stands for

Reply:
I agree that `javascript` (and `js`?) should be added